pkg/shell-operator/operator.go

```diff
@@ -428,7 +428,14 @@ func (op *ShellOperator) TaskHandleHookRun(t task.Task) queue.TaskResult {
 		}
 	}
 
-	metrics, err := taskHook.Run(hookMeta.BindingType, hookMeta.BindingContext, hookLogLabels)
+	usage, metrics, err := taskHook.Run(hookMeta.BindingType, hookMeta.BindingContext, hookLogLabels)
+
+	if usage != nil {
+		log.Infof("Usage: %+v", usage)
+		op.MetricStorage.CounterAdd("{PREFIX}hook_run_usage_sys_ms_total", float64(usage.Sys.Milliseconds()), metricLabels)
```

Comment:
Perhaps, it will be better to convert metrics values to seconds and bytes.
Most metrics use these units in various exporters.

Although we can leave it as is, but I would like to replace short names with full (ms -> milliseconds, kb -> kilobytes).

Reply:
Yes, seconds and bytes are better.